getting-started/setup-building.rst

```diff
@@ -414,27 +414,31 @@ OpenSSL which means that you will not be able to build the ``_ssl`` extension.
 One solution is to install these libraries from a third-party package
 manager, like Homebrew_ or MacPorts_, and then add the appropriate paths
 for the header and library files to your ``configure`` command.  For example,
+with **Homebrew**,
 
-with **Homebrew**::
+For Python 3.10 and newer::
 
-    $ brew install pkg-config openssl@1.1 xz gdbm tcl-tk
+    $ brew install pkg-config openssl xz gdbm tcl-tk
 
-For Python 3.10 and newer::
+and ``configure``::
```

Comment:
```suggestion
Then, for Python 3.10 and newer::
```

To use a more coherent structure, clearly specifying what individual steps are specific to which Python versions, and match with the above.